### DISCUSSION TITLE
How to include theme when project uses CSS, not SCSS?

### DISCUSSION THREAD

**Discussion by JWess** [ng-matero/extensions#184] [Q&A]
I have a project (Angular 14, Material 14, @ng-matero/extensions@14) that uses CSS only, not SCSS. In my `src\styles.css`, I include the default Material theme like this `@import '@angular/material/prebuilt-themes/indigo-pink.css';`. How do I include the theme for MtxDatetimepicker?

**Comment by nzbin** [comment]
Now 15.2.0 has added pre-built themes, I will sync this feature to v14 in the next week.

**Reply by nzbin** [reply]
14.7.0 has already added pre-built themes.
